JavaScript/JavaScript.sublime-syntax

```diff
@@ -1029,7 +1029,7 @@ contexts:
           set:
             - meta_scope: meta.function-call.method.js
             - include: function-call-params
-        - match: \b(process)(?:(\.)(abort|chdir|cwd|exit|getgid|setgid|getuid|setuid|setgroups|getgroups|initgroups|kill|memoryUsage|nextTick|umask|uptime|hrtime))?\b
+        - match: \b(process)(?:(\.)(abort|chdir|cwd|disconnect|exit|[sg]et[e]?gid|[sg]et[e]?uid|send|[sg]etgroups|initgroups|kill|memoryUsage|nextTick|umask|uptime|hrtime))?\b
```

Comment:
the `[e]?` is necessary to match both  `egid` & `euid` variants of `get` and `set` while still being able to match the non `e` variants. I could have done `[sg]et[e]?[gu]id` but I was considering the readability of it at the time.